README.md

```diff
@@ -1,244 +1,271 @@
 
 ```python
-weather_data_ordered.plot(subplots=True, x= 'Formatted Date', y= ['Temperature (C)', 'Humidity'],figsize=(15, 8))
+weather_data_ordered.plot(
+    subplots=True, x='Formatted Date', y=['Temperature (C)', 'Humidity'],
+    figsize=(15, 8))
 ```
 
-13. Now try plotting different variables in the same plot for different years.
+13. Now, try plotting different variables in the same plot for different years.
+
 
 ## Data Summarization and Aggregation
-The previous plots contained a lot of data which make it difficult to understand the evolution of our variables through time. For this reason, one of the things that we can do is grouping the information we have by years and plot the value of the variables year by year. For this, we have created a function that creates a new column in the data frame containing the year, and then groups values by year, computing the average of the variables for each one. 
 
-14. Import the function 
+The previous plots contained a lot of data, which make it difficult to understand the evolution of our variables through time.
+For this reason, one of the things that we can do is group the information we have by year and plot the yearly values.
+To do this, we have written a function in the `utils.py` file, in the same folder as your workshop, that creates a new column in the DataFrame containing the year, and then groups values by year, computing the average of the variables for each one.
+
+14. Import the function from the `utils` module, so you can use it in your script:
 
 ```python
-from utils import (aggregate_by_year)
+from utils import aggregate_by_year
 ```
 
-and use the following command
+15. Use it to aggregate by year and plot the data:
 
- ```python
- weather_data_by_year = aggregate_by_year(weather_data_ordered, 'Formatted Date')
+```python
+weather_data_by_year = aggregate_by_year(
+    weather_data_ordered, date_column='Formatted Date')
 ```
 
-to plot the values of the variables for each year.
+16. Try writing a function in the `utils.py` file that gets the averages of the weather data by month and plots them.
 
-15. Try writing a function in the utils.py file that gets the average of the weather data by month and plot it. 
 
 ## Data Analysis and Interpretation
 
-Now, we want to evaluate the relationships between the variables in our data set. For this, we have written a function in the file 'utils.py' which should be in the same folder of your workshop. 
+Now, we want to evaluate the relationships between the variables in our data set.
+For this, we have written another function in `utils.py`.
 
-16. Import the function from this file, to be able to use it
+17. Import the new function:
 
 ```python
-from utils import (aggregate_by_year, plot_correlations) 
+from utils import aggregate_by_year, plot_correlations
 ```
 
-17. Plot the correlations between the variables using the command 
+18. Plot the correlations between the variables:
```

Comment:
```suggestion
18. Plot the correlations between the variables and view it in the plots pane:
```

Reply:
```suggestion
18. Plot the correlations between the variables and view the figure in the plots pane:
```

Correlations is plural, which it implicitly referred to